code/game/objects/items/emags.dm

```diff
@@ -126,3 +126,31 @@
 			return TRUE
 	to_chat(user, span_warning("[src] is unable to interface with this. It only seems to fit into airlock electronics."))
 	return FALSE
+
+/*
+ * Battlecruiser Access
+ */
+/obj/item/card/emag/battlecruiser
+	name = "battlecruiser coordinates upload card"
+	desc = "An ominous card that contains the location of the station, and when applied to a communications console, \
+	the ability to long-distance contact the Syndicate fleet."
+	icon_state = "battlecruisercaller"
+	worn_icon_state = "battlecruisercaller"
+	///whether we have called the battlecruiser
+	var/used = FALSE
+
+/obj/item/card/emag/battlecruiser/proc/use_charge(mob/user)
+	used = TRUE
+	to_chat(user, span_boldwarning("You use [src], and it interfaces with the communication console. No going back..."))
+
+/obj/item/card/emag/battlecruiser/examine(mob/user)
+	. = ..()
+	. += span_notice("It can only be used on the communications console.")
+
+/obj/item/card/emag/battlecruiser/can_emag(atom/target, mob/user)
+	if(used)
+		to_chat(user, span_warning("[src] is used up."))
```

Comment:
`can_emag` is a name that implies purity, and thus should not perform any side effects.

Find a better place to insert this chat message, or rename the proc to `try_emag` or similar.

Reply:
@tralezab